public/app/features/alerting/unified/AlertGroups.tsx

```diff
@@ -68,6 +79,13 @@ const AlertGroups = () => {
           {error.message || 'Unknown error'}
         </Alert>
       )}
+
```

Comment:
Why we don't use `GrafanaAlertmanagerDeliveryWarning` in here?

Reply:
Mainly due to a different warning message. Alert groups are read-only, so mentioning `changing configuration` might be confusing

Reply:
ok! make sense. Then we could say someting like: 
```
Grafana is configured to send alerts to the external Alertmanagers only.
External alert manager grouping will be the only one applied.
```